src/components/chip-set/chip-set.tsx

```diff
@@ -276,6 +287,7 @@ export class ChipSet {
             disabled: this.disabled || this.readonly,
             'mdc-text-field--with-trailing-icon': true,
         };
+
```

Comment:
Please avoid whitespace changes in commits with functional changes. I'm guessing this just happened, but if you meant to include it, please put it in a separate commit.